Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Spawnable/SpawnableUtils.cpp

```diff
@@ -47,6 +165,122 @@ namespace AzToolsFramework::Prefab::SpawnableUtils
+        AzToolsFramework::Prefab::PrefabConversionUtils::EntityAliasSpawnableLoadBehavior loadBehavior,
+        uint32_t tag,
+        AzToolsFramework::Prefab::PrefabConversionUtils::PrefabProcessorContext& context)
+    {
+        using namespace AzToolsFramework::Prefab::PrefabConversionUtils;
+
+        AliasPath alias = source.GetAliasPathRelativeToInstance(entity);
+        if (!alias.empty())
+        {
+            auto&& [replacement, storedAliasType] = Internal::ApplyAlias(source, entity, aliasType);
+
+            AZ::Entity* result = replacement.get();
+            target.AddEntity(AZStd::move(replacement), alias.Filename().Native());
+
+            EntityAliasStore store;
+            store.m_aliasType = storedAliasType;
+            store.m_source.emplace<EntityAliasPrefabLink>(AZStd::move(sourcePrefabName), AZStd::move(alias));
+            store.m_target.emplace<EntityAliasPrefabLink>(
+                AZStd::move(targetPrefabName), target.GetAliasPathRelativeToInstance(result->GetId()));
+            store.m_loadBehavior = loadBehavior;
+            store.m_tag = tag;
+            context.RegisterSpawnableEntityAlias(AZStd::move(store));
+
+            return result;
+        }
+        else
+        {
+            AZ_Assert(false, "Entity with id %zu was not found in the source prefab.", static_cast<AZ::u64>(entity));
+            return nullptr;
+        }
+    }
+
+    AZ::Entity* CreateEntityAlias(
+        AZStd::string sourcePrefabName,
+        AzToolsFramework::Prefab::Instance& source,
+        AzFramework::Spawnable& target,
+        AZ::EntityId entity,
+        AzToolsFramework::Prefab::PrefabConversionUtils::EntityAliasType aliasType,
+        AzToolsFramework::Prefab::PrefabConversionUtils::EntityAliasSpawnableLoadBehavior loadBehavior,
+        uint32_t tag,
+        AzToolsFramework::Prefab::PrefabConversionUtils::PrefabProcessorContext& context)
+    {
+        using namespace AzToolsFramework::Prefab::PrefabConversionUtils;
+
+        AliasPath alias = source.GetAliasPathRelativeToInstance(entity);
+        if (!alias.empty())
+        {
+            auto&& [replacement, storedAliasType] = Internal::ApplyAlias(source, entity, aliasType);
+
+            AZ::Entity* result = replacement.get();
+            target.GetEntities().push_back(AZStd::move(replacement));
+            
+            EntityAliasStore store;
+            store.m_aliasType = storedAliasType;
+            store.m_source.emplace<EntityAliasPrefabLink>(AZStd::move(sourcePrefabName), AZStd::move(alias));
+            store.m_target.emplace<EntityAliasSpawnableLink>(target, result->GetId());
+            store.m_tag = tag;
+            store.m_loadBehavior = loadBehavior;
+            context.RegisterSpawnableEntityAlias(AZStd::move(store));
+
+            return result;
+        }
+        else
+        {
+            AZ_Assert(false, "Entity with id %zu was not found in the source prefab.", static_cast<AZ::u64>(entity));
```

Comment:
nit: the cast should be to `size_t` here to match the print specifier (or use `PRIx64` in the format string)